src/main/resources/anniv5Resources/localization/eng/contentcreatorpack/Powerstrings.json

```
@@ -0,0 +1,44 @@
+{
+  "${ModID}:HuttsGambleEnergyPower": {
+    "NAME": "Hutts' Energized",
+    "DESCRIPTIONS": [
+      "If you do not receive unblocked attack damage this turn, gain #b",
+      " Energy next turn."
+    ]
+  },
+  "${ModID}:HuttsGambleDrawPower": {
+    "NAME": "Hutts' Draw",
+    "DESCRIPTIONS": [
+      "If you do not receive unblocked attack damage this turn, draw #b",
+      " card next turn.",
+      " cards next turn."
+    ]
+  },
+  "${ModID}:RetromationPower": {
+    "NAME": "Retromation",
+    "DESCRIPTIONS": [
+      "At the start of your turn, gain #b",
+      " card",
+      " cards",
+      " from a Pack not available to your draft. NL ",
```

Same common as for the card, maybe use "in".
```suggestion
      " from a Pack not available in your draft. NL ",
```